tutorials/zulip/index.mdx

```diff
@@ -78,14 +78,14 @@ More than 90 integrations are available for Zulip, including GitHub, Jira, Zende
     <Message type="tip">
       The link is a secure one-time-use link. If you have lost it or if you need another one later, you can generate a new one by running `/home/zulip/deployments/current/manage.py generate_realm_creation_link` in the Zulip main directory.
     </Message>
-6. Open your web browser and point it to your Zulip activation link. Enter your email address to create new organization. Confirm by clicking **Create Organization**:
+6. Open your web browser and point it to your Zulip activation link. Enter your email address to create a new organization. Confirm by clicking **Create Organization**:
     <Lightbox src="scaleway-zulip_new_orga.webp" alt="" />
 7. Complete the registration by providing your organization's name and additional information, such as a secure password. Once completed, you see Zulip's main screen:
     <Lightbox src="scaleway-zulip_dashboard.webp" alt="" />
 
 ## Configuring outgoing emails
 
-Your Zulip is now running, however, it cannot send any email notifications in its current state. To send emails we configure a transactional email provider using SMTP which will take care of the delivery of all notifications Zulip sends out. In this tutorial we use the free-tier offer from [SendInBlue](https://www.sendinblue.com/), but you are free to use any transactional email provider. Be aware that using SMTP credentials of an inbound email provider (like Google Mail) may flag your mails as spam and have an impact on the delivery reliability of the notifications.
+Your Zulip is now running, however, it cannot send any email notifications in its current state. To send emails we configure a transactional email provider using SMTP which will take care of the delivery of all notifications Zulip sends out. In this tutorial, we use the free-tier offer from [SendInBlue](https://www.sendinblue.com/), but you are free to use any transactional email provider. Be aware that using the SMTP credentials of an inbound email provider (like Google Mail) may flag your emails as spam and have an impact on the delivery reliability of the notifications.
```

Comment:
Maybe we need a guideline on whether we write SPAM or spam because I've seen it both ways in our doc